File: 100-Days-Series/Day 94/ReplaceNegativeNumberWithZero.py

# Function to replace negative numbers with zero
def get_replace_negative_with_zero(arr):
    for i in range(0, len(arr)):
        if arr[i] < 0 :
            arr[i] = 0


# Let's Start Here
arr=[-1, 2, -3, 4, -5]
print('Original Array')
print(arr)
get_replace_negative_with_zero(arr)
print('Updated Array')
print(arr)

File: 100-Days-Series/Day 94/ReplaceNegativeNumberWithZero.js

// Function to replace negative numbers with zero.
const getReplaceNegativeNumberWithZero=(arr)=>{
    for(let i=0;i<arr.length;i++){
        if(arr[i]<0){
            arr[i]=0;
        }
    }
}
// Let's Start Here
const arr=[-1, 2, -3, 4, -5];
console.log("Original Array");
console.log(arr);
getReplaceNegativeNumberWithZero(arr);
console.log("Updated Array");
console.log(arr);